apps/chat/src/components/Chat/MessageAttachment.tsx

```diff
@@ -281,7 +296,10 @@ export const MessageAttachment = ({ attachment, isInner }: Props) => {
         </button>
       </div>
       {isOpenable && isOpened && (
-        <div className="relative mt-2 h-auto w-full overflow-hidden p-3 pt-4 text-sm duration-200">
+        <div
+          className="relative mt-2 h-auto w-full overflow-hidden p-3 pt-4 text-sm duration-200"
+          ref={anchorRef}
```

Comment:
As I know, React recommends to use only 1 `ref` for a particular element. Here `anchorRef` is applied for 2 elements

Reply:
these elements can exist together and it's main idea

Reply:
Ohh. there it's mistake, but the comment is correct for `ChatMessageContent`